### PR TITLE
The example of quantization with accuracy control for OpenVINO backend

### PR DESCRIPTION
**Pull request by alexsu52**
### Changes

The example of quantization with accuracy control for OpenVINO backend

### Reason for changes

Demonstrate how to use Post-Training Quantization with accuracy control API.
`nncf.quantize_with_accuracy_control()` 

### Related tickets

ref: 98687

### Tests

TBD

### CODE REVIEW

File: examples/post_training_quantization/openvino/quantize_with_accuracy_control/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Post-Training Quantization of Anomaly Classification OpenVINO model with control of accuracy metric
+
+The anomaly detection domain is one of the domains in which models are used in scenarios where the cost of model error is high and accuracy cannot be sacrificed for better model performance. For such cases, it is necessary to use quantization methods with accuracy control where the maximum accuracy drop can be specified as an argument of the method.
+
+This example demonstrates how to quantize [SFTPM](https://huggingface.co/alexsu52/stfpm_mvtec_capsule) OpenVINO model from [Anomalib](https://github.com/openvinotoolkit/anomalib) using Post-Training Quantization with accuracy control API from Neural Network Compression Framework (NNCF).
```
Comment (AlexKoff88) [inline examples/post_training_quantization/openvino/quantize_with_accuracy_control/README.md:5]: Can we have the full name of the model at the first mentioning?

Comment (AlexKoff88) [reply examples/post_training_quantization/openvino/quantize_with_accuracy_control/README.md:5]: Like "Student-Teacher Feature Pyramid Matching (SFTPM)"

**Comment by AlexKoff88** [comment]
I ok with the sample. The API usage is demonstrated well, the rest is a matter of opinion.

File: examples/post_training_quantization/openvino/quantize_with_accuracy_control/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Post-Training Quantization of Anomaly Classification OpenVINO model with control of accuracy metric
+
+The anomaly detection domain is one of the domains in which models are used in scenarios where the cost of model error is high and accuracy cannot be sacrificed for better model performance. For such cases, it is necessary to use quantization methods with accuracy control where the maximum accuracy drop can be specified as an argument of the method.
+
+This example demonstrates how to quantize [Student-Teacher Feature Pyramid Matching (SFTPM)](https://huggingface.co/alexsu52/stfpm_mvtec_capsule) OpenVINO model from [Anomalib](https://github.com/openvinotoolkit/anomalib) using Post-Training Quantization with accuracy control API from Neural Network Compression Framework (NNCF).
```
Comment (l-bat) [inline examples/post_training_quantization/openvino/quantize_with_accuracy_control/README.md:5]: Why not `STFPM`?

**Comment by alexsu52** [comment]
run onnx pre-commit tests

**Pull request merged**